resources/profiles/Creality.json

```diff
@@ -410,22 +278,6 @@
 			"name": "0.36mm SuperChunky @Creality CR-6 0.6",
 			"sub_path": "process/0.36mm SuperChunky @Creality CR-6 0.6.json"
 		},
-		{
-			"name": "0.36mm Chunky @Creality Ender5Pro (2019) 0.5",
```

Comment:
@bistory 
I noticed you have deleted all Creality Ender5Pro (2019) profiles, is this intentional?

Reply:
Not at all 😅

Reply:
I think it's due to a bad merge, I will fix that ;-)